### DISCUSSION TITLE
Localized quotation marks

### DISCUSSION THREAD

**Discussion by cbrnr** [quarto-dev/quarto-cli#4992] [Q&A] [answered]
I'm writing a Quarto document with `lang: de` in my YAML header. However, I noticed that quotation marks in the rendered HTML (probably also in other formats) are still English. Is it possible to get German quotation marks? German is just an example for any non-English language, so I'd be interested to learn how to enable non-English quotation marks in general.

**Comment by cderv** [comment, marked as answer]
I think this is not built-in Pandoc and we did not do anything specific around this. Current advice for Pandoc is to use a Lua filter (https://github.com/pandoc/lua-filters/tree/master/pandoc-quotes.lua) to do this. Details are given in their [man page](https://github.com/pandoc/lua-filters/blob/master/pandoc-quotes.lua/man/pandoc-quotes.lua.md)

It should work with Quarto, even if it has not been ported as an extension in https://github.com/pandoc-ext/

Related issue in upstream Pandoc https://github.com/jgm/pandoc/issues/2620

@dragonstyle do you think we could bring this type of filter as a built in feature in Quarto ? Maybe as an extension it is ok though

**Reply by cbrnr** [reply]
Thanks! I saw the custom Lua filter solution, but thought that maybe I'm missing something that's perhaps more integrated. I'd love to see this either as a built-in feature or extension!

**Reply by cderv** [reply]
Maybe we have already something I am not aware - @dragonstyle will tell us. My understanding is that we rely mainly on Pandoc feature for localization.

**Reply by dragonstyle** [reply]
This does seem correct- I think it would be a great extension to support something more integrated, but right now I think taking this Pandoc approach seems correct. I looked briefly at csquotes or other LaTeX packages but couldn't find the way to get them to play nicely with our LaTeX output.
